CONTRIBUTING.md

```diff
@@ -2,31 +2,112 @@
 
 ## How to contribute
 
-Before kicking the ball, please **do NOT report security vulnerabilities in public issues**! Instead, report with the CloudFoundry Foundation team first <security@cloudfoundry.org> and give us some time to fix it in a fashion time matter before disclosing it to the public. For more information check the CloudFoundry Security [page](https://www.cloudfoundry.org/security/).
+Before kicking the ball, please **do NOT report security vulnerabilities in public issues**! Instead, please report with the
+CloudFoundry Foundation team first <security@cloudfoundry.org> and give us some time to fix it in a fashion time matter
+before disclosing it to the public. 
+For more information, check the CloudFoundry Security [page](https://www.cloudfoundry.org/security/).
 
-When contributing to this repository, please first discuss the changes through an existent Github issue with the main contributers.
+When contributing to this repository, please first discuss the changes through an existing Github issue with the main
+contributors.
 
 ## Issues tracker
 
-We're commited to have short and accurate templates for different type issues, in order to gather the required information to start a conversation.
+We're commited to have short and accurate templates for different type issues in order to gather the required
+information to start a conversation.
 
 Once again, feel free to contribute to improve them by opening a pull request.
 
 ### How to report a bug
 
-Start by searching [bugs][1] for some similiar issues and/or extra information. If your search doesn't bring you any help then open an issue by selecting the issue type "Bug" and fill the template accurately as possible.
+Start by searching [bugs][1] for some similiar issues and/or extra information. If your search doesn't bring you any
+help, then open an issue by selecting the issue type "Bug" and fill the template accurately as possible.
 
 ### How to suggest a feature/enhancement
 
-If you find yourself wishing for a feature/enhancement that does not exist yet in KubeCF start by run a quick [search][2] - you may not be alone! If you ain't any luck, then go ahead and open a new issue by selecting the "Feature" issue type and answer some needed questions.
+If you find yourself wishing for a feature/enhancement that does not exist yet in KubeCF, start by run a
+quick [search][2] - you may not be alone! If you aren't having any luck, then go ahead and open a new issue by selecting the
+"Feature" issue type and answer some needed questions.
 
 ## Code review process
 
-The core team looks to Pull Requests on a regular basis and in a best effort.
+The core team looks at Pull Requests on a regular basis on a best-effort basis.
+
+## How are Github issues handled
+
+### Queues
+
+1. icebox: contain new issues along with issues that are waiting for grooming and not read to start working yet. Some issues
+may be closed or rejected.
+
```

Comment:
Suggest rewording to:

"New issues that require further grooming. Note that some issues may be closed or rejected.

Reply:
IMHO "Contains new issues that ..." (note the trailing `s` on `Contains`, vs original, and capitalization).